metasyn/metaframe.py

```diff
@@ -53,11 +53,11 @@ def n_columns(self) -> int:
     @classmethod
     def fit_dataframe(
             cls,
-            df: pl.DataFrame,
-            spec: Optional[dict[str, dict]] = None,
-            dist_providers: Union[str, list[str], BaseDistributionProvider,
-                                  list[BaseDistributionProvider]] = "builtin",
-            privacy: Optional[BasePrivacy] = None,
+            df: Optional[Union[pl.DataFrame, pd.DataFrame]],
```

Comment:
So we support either now? I like that but I don't want pandas as a dependency!

This change should also be in docs

Reply:
We have always supported both technically, at least a lot of our test suite checks the results for both. I'm okay with trying to remove pandas as a dependencies and relying on polars to do the heavy lifting. But I think we should do that in a separate PR. Do you agree?